Makefile

```diff
@@ -74,7 +74,8 @@ fmt format:
 
 .PHONY: mypy
 mypy:
-	mypy --strict aiohttp tests
+	mypy --strict aiohttp
+	mypy tests
```

Comment:
Can't you control this via the config?

Reply:
Solved by adding pytest to lint.txt requirements file